Sources/Orbit/Components/InputField.swift

```diff
@@ -96,6 +125,25 @@ public struct InputField: View {
                 }
         }
     }
+
+    @ViewBuilder var unsecuredField: some View {
+        textField
+            .disabled(true)
+            .customOnChange(of: value, perform: { _ in
+                secureModel.showSecureText = false
+            })
+            .opacity(secureModel.showSecureText ? 1 : 0)
+            .animation(.easeOut(duration: 0.2), value: secureModel.showSecureText)
+    }
+
+    @ViewBuilder var securedSuffix: some View {
+        Icon.Content.icon(secureModel.showSecureText ? .visibilityOff : .visibility, size: .default)
```

Comment:
can be just `Icon(secureModel.showSecureText ? .visibilityOff : .visibility, size: .default)`

Reply:
@PavelHolec updated. Please review all opened threads, they should be done as well 🙏